analysis/sample/sample_data_set.textproto

```diff
@@ -0,0 +1,30 @@
+sequences: {
+  page_views: {
+    contents: {
+      font_name: "Roboto-Regular.ttf"
```

Comment:
future consideration: it's quite plausible to encounter multiple different versions of a file. Maybe we need url of font or even a checksum?

Reply:
Will be fixed in an up comming PR.

Reply:
The intent is that sort of concern is handled during the preparation of the input data set.  The input data set is both the set of font files and the page view sequences. If there's multiple versions of the same font then during input data set preparation the different versions should be fetched and given unique names and then the page view sequences should refer to the different versions by the appropriate names.